scripts/fetch-asyncapi-example.js

```diff
@@ -6,7 +6,7 @@ const unzipper = require('unzipper');
 const path = require('path');
 const parser = require('@asyncapi/parser');
 
-const SPEC_EXAMPLES_ZIP_URL = 'https://github.com/asyncapi/spec/archive/refs/tags/v2.3.0.zip';
+const SPEC_EXAMPLES_ZIP_URL = 'https://github.com/asyncapi/spec/archive/refs/heads/master.zip';
```

Comment:
yeah, for reasons like https://github.com/asyncapi/spec/pull/873 I think we need to always point to master.

Reply:
as a result we can close:
- https://github.com/asyncapi/cli/pull/213
- https://github.com/asyncapi/cli/issues/205